pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -128,345 +154,465 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 			return ctrl.Result{}, nil
 		}
 
-		log.Error(err, errGetES)
+		log.Error(err, logErrorGetES)
 		syncCallsError.With(resourceLabels).Inc()
-
-		return ctrl.Result{}, err
-	}
-
-	// See https://github.com/external-secrets/external-secrets/issues/3604
-	// We fetch the ExternalSecret resource above, however the status subresource is inconsistent.
-	// We have to explicitly fetch it, otherwise it may be missing and will cause
-	// unexpected side effects.
-	err = r.SubResource("status").Get(ctx, &externalSecret, &externalSecret)
-	if err != nil {
-		log.Error(err, "failed to get status subresource")
 		return ctrl.Result{}, err
 	}
 
-	timeSinceLastRefresh := 0 * time.Second
-	if !externalSecret.Status.RefreshTime.IsZero() {
-		timeSinceLastRefresh = time.Since(externalSecret.Status.RefreshTime.Time)
-	}
-
 	// skip reconciliation if deletion timestamp is set on external secret
-	if externalSecret.DeletionTimestamp != nil {
-		log.Info("skipping as it is in deletion")
+	if !externalSecret.GetDeletionTimestamp().IsZero() {
+		log.V(1).Info("external secret is marked for deletion, skipping reconciliation")
```

Comment:
Why the changes to `V(1)`?

Reply:
Because this can happen whenever an ExternalSecret CRD is deleted, it's not useful information for the user, because it's fully expected.

It's just the controller realizing that it should not reconcile that specific ExternalSecret anymore, because it's about to be deleted.